render.c

```diff
@@ -228,19 +259,46 @@ static int render_notification(cairo_t *cairo, struct mako_state *state,
 
 	if (icon != NULL) {
 		// Render icon
-		double xpos = offset_x + style->border_size +
-			(text_x - icon->width) / 2;
-		double ypos = offset_y + style->border_size +
-			(notif_height - icon->height - border_size) / 2;
+		double xpos = -1;
+		double ypos = -1;
+		switch (style->icon_location) {
+		case MAKO_ICON_LOCATION_LEFT:
+			xpos = offset_x + style->border_size +
+				(text_x - icon->width) / 2;
+			ypos = offset_y + style->border_size +
```

Comment:
Can we make this a little bit easier to read by storing this value in a `ypos_center` variable before the switch? Same for `xpos_center`.